SwiftRuntimeLibrary.Mac/SwiftRuntimeLibrary.Mac.csproj

```diff
@@ -1,311 +1,296 @@
+      <Link>UnsafeMutableRawBufferPointer.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\UnsafeRawBufferPointer.cs">
+      <Link>UnsafeRawBufferPointer.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftHasher.cs">
+      <Link>SwiftHasher.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftDate.cs">
+      <Link>SwiftDate.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftFoundationConstants.cs">
+      <Link>SwiftFoundationConstants.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\EveryProtocol.cs">
+      <Link>EveryProtocol.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\ExistentialContainers.cs">
+      <Link>ExistentialContainers.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\BaseProxy.cs">
+      <Link>BaseProxy.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftMarshal\SwiftTypeNameAttribute.cs">
+      <Link>SwiftMarshal\SwiftTypeNameAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftTypeRegistry.cs">
+      <Link>SwiftTypeRegistry.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftMarshal\SwiftProtocolConformanceDescriptor.cs">
+      <Link>SwiftMarshal\SwiftProtocolConformanceDescriptor.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftMarshal\SwiftProtocolWitnessTable.cs">
+      <Link>SwiftMarshal\SwiftProtocolWitnessTable.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftMarshal\SwiftAssociatedTypeDescriptor.cs">
+      <Link>SwiftMarshal\SwiftAssociatedTypeDescriptor.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\BaseAssociatedTypeProxy.cs">
+      <Link>BaseAssociatedTypeProxy.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftNativeInstance.cs">
+      <Link>SwiftNativeInstance.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftNativeObject.cs">
+      <Link>SwiftNativeObject.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftNativeValueType.cs">
+      <Link>SwiftNativeValueType.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\ISwiftValueType.cs">
+      <Link>ISwiftValueType.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftMarshal\SwiftThrowsAttribute.cs">
+      <Link>SwiftMarshal\SwiftThrowsAttribute.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\ISwiftIteratorProtocol.cs">
+      <Link>ISwiftIteratorProtocol.cs</Link>
+    </Compile>
+    <Compile Include="..\SwiftRuntimeLibrary\SwiftIteratorProtocolProxy.cs">
+      <Link>SwiftIteratorProtocolProxy.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="SwiftMarshal\" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Xamarin.Mac" />
```

Comment:
This doesn't look right, you shouldn't be referencing Xamarin.Mac.dll anymore (you should automatically get a reference to Microsoft.macOS.dll because TargetFramework=net7.0-macos).

Reply:
This got managed in a later PR.